src/detect.h

```diff
@@ -424,6 +424,8 @@ typedef struct DetectEngineAppInspectionEngine_ {
     uint8_t id;     /**< per sig id used in state keeping */
     bool mpm;
     bool stream;
+    // will match on a NULL buffer (only negated content)
```

Comment:
can you use doxygen style comments for structs

Reply:
/** ok  */